dbus/src/message.rs

```diff
@@ -59,6 +59,20 @@ impl Message {
         Message { msg: ptr}
     }
 
+    /// Creates a new message that is a replica of this message, but without a serial.
+    ///
+    /// May fail if out of memory or file descriptors.
+    pub fn duplicate(&self) -> Result<Self, String> {
+        let ptr = unsafe {
```

Comment:
Is there a reason this function is not called `copy`?

Reply:
Calling it `copy` would suggest to me it is something related to `std::marker::Copy`, which it is definitely not.